soc/bflb/common/e24/intc_clic.S

```diff
@@ -9,19 +9,6 @@
  */
 
 #include <zephyr/arch/cpu.h>
-#include "clic.h"
```

Comment:
I'm wondering why we still need this file after all these removals.

Reply:
You are right, it's not needed. But it triggers the bug this PR is trying to fix when using the riscv-privileged .S... Which I didnt catch at first and thought it was because it needed it defined.